plugins/modules/storage/netapp/na_cdot_license.py

```diff
@@ -31,11 +31,13 @@
     description:
     - Remove licenses that have no controller affiliation in the cluster.
     type: bool
+    default: false
```

Comment:
this file is deprecated.

Reply:
Yes. It still exists (and will keep existing on the stable-1 branch), and I don't want sanity tests start randomly once the ansible-test PR is merged.